src/main/resources/assets/minecolonies/lang/manual_en_us.json

```diff
@@ -2127,5 +2127,94 @@
+  "minecolonies:student.job.desc": "Reads books to increase their intelligence. High intelligence allows learning other skills faster.",
+  "minecolonies:archertraining.job.desc": "Archer Trainee. Practices distance fighting.",
+  "minecolonies:combattraining.job.desc": "Knight Trainee. Practices close quarter fighting.",
+  "minecolonies:sawmill.job.desc": "Provides the colony with most wood based items.",
+  "minecolonies:blacksmith.job.desc": "Provides the colony with most tools, armor and weapons.",
+  "minecolonies:stonemason.job.desc": "Provides most stone variants.",
+  "minecolonies:stonesmeltery.job.desc": "Provides the stone variants that require smelting and charcoal",
+  "minecolonies:crusher.job.desc": "Crushes cobble to gravel, gravel to sand and sand to clay.",
+  "minecolonies:sifter.job.desc": "Sifts through a variety of materials to find goodies.",
+  "minecolonies:florist.job.desc": "Plants a large variety of flowers for the colony.",
+  "minecolonies:enchanter.job.desc": "Gathers mana by observing others, enchants colonist tools and creates enchanting books.",
+  "minecolonies:researcher.job.desc": "Researches the topics the player selected.",
+  "minecolonies:healer.job.desc": "Heals wounded entities and provides cures for the sick.",
+  "minecolonies:pupil.job.desc": "Sits around and learns from the material provided by the teacher.",
+  "minecolonies:teacher.job.desc": "Sits with the kids and provides material.",
+  "minecolonies:dyer.job.desc": "Handles most of the dying that is required in the colony.",
+  "minecolonies:glassblower.job.desc": "Provides most glass based items.",
+  "minecolonies:mechanic.job.desc": "Handles redstone recipes and things other workers can't handle.",
+  "minecolonies:fletcher.job.desc": "Handles leather, string and wool based recipes.",
+  "minecolonies:planter.job.desc": "Provides sugarcane, cactus and bamboo to the colony.",
+  "minecolonies:rabbitherder.job.desc": "Handles a flock of rabbits.",
+  "minecolonies:concretemixer.job.desc": "Creates concrete powder and actual concrete",
+  "minecolonies:beekeeper.job.desc": "Administrates several beehives",
+  "minecolonies:cookassistant.job.desc": "Handles food based recipes which the main cook can't handle.",
+  "minecolonies:smelter.job.desc": "Smelts raw ores to ingots and breaks ore blocks into raw ore.",
+  "minecolonies:druid.job.desc": "Support type unit. Buffs friendlies and nerfs enemies.",
+
+  "minecolonies:builder.skills.desc": "Influence the placement and breaking speed.",
+  "minecolonies:deliveryman.skills.desc": "Influences the walking speed and carrying capacity.",
+  "minecolonies:miner.skills.desc": "Influence the placement and breaking speed.",
+  "minecolonies:quarrier.skills.desc": "Influence the placement and breaking speed.",
+  "minecolonies:lumberjack.skills.desc": "Influences the breaking and walking speed.",
+  "minecolonies:farmer.skills.desc": "Influences the placement, harvesting and context switching speed.",
```

Comment:
What does "context switching speed" mean? Is that switching between harvesting and planting?

Reply:
Switching between tasks and fields